ui-tests/BrowserView.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { shallow } from "enzyme"
+import * as React from "react"
+import { shallowToJson } from "enzyme-to-json"
+
+import { Event } from "oni-types"
+import {
+    BrowserView,
+    IBrowserViewProps,
+    IBrowserViewState,
+} from "../browser/src/Services/Browser/BrowserView"
+import { Configuration } from "../browser/src/Services/Configuration"
+
+const mockEvent = new Event<void>()
+
+// Using the disable life cycle methods here as the CDM calls sneak which is
+// an external dependency I'm not trying to test here
+
+describe("<BrowserView /> Tests", () => {
```

Comment:
Nice, thanks for adding a test to cover this component, @Akin909 !

Reply:
👍 think theres still quite a bit more testing that could be added there I can look at in other PRs